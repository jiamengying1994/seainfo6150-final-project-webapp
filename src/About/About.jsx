import React from 'react';
import style from './About.module.css';

const About = () => (
  <div className={style.aboutpage}>

    <div className = {style.about_text}>
      <h1>About us</h1>
      <p>Mygallery is founded since 2014. Selling paintings by modern artists is the main concept of Mygallery.</p >
      <p>We give painting lovers an opportunity to touch both the masterpieces by recognized brilliant artists and the works by beginning artists.</p >
  <p>Everyone can buy a painting directly by calling directly or left information, and often cheaper than in usual galleries. </p >
    </div>

  </div>
  );

export default About;
