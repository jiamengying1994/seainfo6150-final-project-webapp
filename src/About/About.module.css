@media(min-width:300px){

.aboutpage {
      height: 40rem;
      width: 100%;
      display: flex;
      align-items: center;
      justify-content: center;
}
.about_text{
     width: 80%;
     height: auto;
     margin: 0.1em;
     color: white;
     background: hsla(0, 5%, 10%, .9);
     text-align: left;
     position: absolute;
     padding-left: 5%;
     padding-right: 5%;
     padding-top: 2%;
     padding-bottom: 2%;
     z-index: 1;
     font-size: 10px;
     display: block;
}
}

@media(min-width:600px){

.aboutpage {
      height: 40rem;
      width: 100%;

}
.about_text{
     font-family: Courier New;
     width: 80%;
     height: 40%;
     margin: 1em;
     background: hsla(0, 5%, 10%, .9);
     color: white;
     text-align: left;
     position: absolute;
     padding-left: 5%;
     padding-right: 5%;
     padding-top: 2%;
     padding-bottom: 2%;
     z-index: 1;
     font-size: 1em;
     display: block;
}
}



@media(min-width:900px){

.aboutpage {
      height: 40rem;
      width: 100%;

}
.about_text{
     font-family: Courier New;
     width: 80%;
     height: 60%;
     margin: 1em;
     background: hsla(0, 5%, 10%, .9);
     color: white;
     text-align: left;
     position: absolute;
     padding-left: 5%;
     padding-right: 5%;
     padding-top: 2%;
     padding-bottom: 2%;
     z-index: 1;
     font-size: 1.5em;
     display: block;
}
}