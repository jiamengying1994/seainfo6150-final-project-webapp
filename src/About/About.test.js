import React from "react";
import { render } from "@testing-library/react";
import About from "./About.jsx";

it("renders the about component correctly", () => {
  const { container } = render(<About />);
  expect(container).toMatchSnapshot();
});