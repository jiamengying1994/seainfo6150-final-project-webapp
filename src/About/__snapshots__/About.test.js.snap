// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the about component correctly 1`] = `
<div>
  <div
    class="aboutpage"
  >
    <div
      class="about_text"
    >
      <h1>
        About us
      </h1>
      <p>
        Mygallery is founded since 2014. Selling paintings by modern artists is the main concept of Mygallery.
      </p>
      <p>
        We give painting lovers an opportunity to touch both the masterpieces by recognized brilliant artists and the works by beginning artists.
      </p>
      <p>
        Everyone can buy a painting directly by calling directly or left information, and often cheaper than in usual galleries. 
      </p>
    </div>
  </div>
</div>
`;
